src/utils/getFontValue.ts

```diff
@@ -78,6 +78,7 @@ function useWidth() {
 
   return (
     keys.reduce((output, key) => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Is something not properly used here? Might need a refactor

Reply:
it is the same code as from the MUI template, so I kinda trust it. Also haven't experienced any problems with it , so I suggest to keep it like this.